environment.go

```diff
@@ -288,7 +296,7 @@ func (e *envState) initSettings(path string) error {
 func (e *envState) initDB(ctx context.Context, settings DBSettings) error {
 	opts := options.Client().ApplyURI(settings.Url).SetWriteConcern(settings.WriteConcernSettings.Resolve()).
 		SetReadConcern(settings.ReadConcernSettings.Resolve()).
-		SetConnectTimeout(5 * time.Second).SetMonitor(apm.NewLoggingMonitor(ctx, time.Minute, apm.NewBasicMonitor(nil)).DriverAPM())
+		SetConnectTimeout(5 * time.Second).SetMonitor(otelmongo.NewMonitor())
```

Comment:
What happens here if we have the tracer disabled?

Reply:
If we never set up a tracer provider it will default to the [NoopTracerProvider](https://pkg.go.dev/go.opentelemetry.io/otel/api/trace#NoopTracerProvider) and the traces/spans will go nowhere.